src/main/java/com/powsybl/openloadflow/network/impl/LfGeneratorImpl.java

```diff
@@ -133,4 +139,20 @@ public void updateState() {
                 .setP(-targetP)
                 .setQ(Double.isNaN(calculatedQ) ? -generator.getTargetQ() : -calculatedQ);
     }
+
+    @Override
+    public void setRemoteTargetQ(double targetQ) {
+        setReactivePowerControl(getGenerator().getRegulatingTerminal(), targetQ);
```

Comment:
I am unsure this is what is meant with the targetQ option passed to the GeneratorAction from powsybl-core. Can someone confirm this or give a pointer to what is meant?